components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -385,6 +386,13 @@ private CharonException resolveError(UserStoreException e, String defaultMsg) {
         return new CharonException(defaultMsg, e);
     }
 
+    private void handleErrorsOnRoleNamePolicy(Throwable e) throws BadRequestException {
+
+        if (e instanceof UserStoreException && (e.getMessage().contains(ERROR_CODE_INVALID_ROLE_NAME))) {
```

Comment:
will this be null safe?
 `e.getMessage().contains(ERROR_CODE_INVALID_ROLE_NAME)`